src/platform/packages/shared/kbn-storage-adapter/src/index_adapter/integration_tests/index.test.ts

```diff
@@ -43,8 +39,7 @@ const createLoggerMock = (): jest.Mocked<Logger> => {
 
 describe('StorageIndexAdapter', () => {
   let esServer: TestElasticsearchUtils;
-  let esClient: ElasticsearchClient;
-  let kibanaServer: TestKibanaUtils;
```

Comment:
We don't need to start Kibana to get an ES client, so removed it for now. Also refactored this test to avoid starting the ES server more than once and rely on `client.cleanup()` for a clean test env.